File: Python/Manual 03/Question01.py
n = int(input("Enter Upper Half Rows: "))

if n < 0:
    print("Invalid Input")
else:
    i = n
    while i > 0:
        print("*" * i)
        i -= 1

    i = 1
    while i <= n:
        print("*" * i)
        i += 1

File: JavaScript/Manual 03/Question01.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter Number of Rows: ", function (rows) {
  rows = parseInt(rows);
  for (let i = rows; i > 0; i--) {
    console.log("*".repeat(i));
  }
  for (let i = 2; i <= rows; i++) {
    console.log("*".repeat(i));
  }
  rl.close();
});
